DTS-STN/Service-Canada-Labs
# cleaning up pipeline files

# No related task

Removing some unused pipeline files

## Test Instructions
1. Verify that all of [our essential pipelines](https://dev.azure.com/VP-BD/ENDC-DECD-Builds/_build) still exist and run.
